doc-content/enterprise-only/optimizer/optimizer-running-webexamples-proc.adoc

```diff
@@ -0,0 +1,25 @@
+[id='optimizer-running-webexamples-proc']
+= Running the web examples
+Besides the GUI examples, {PRODUCT} also includes a set of web examples for {PLANNER}. The web examples include:
+
+* Vehicle routing: calculating the shortest possible route to pick up all items required for a number of different customers using either http://leafletjs.com/[Leaflet] or http://google.com/maps[Google Maps] visualizations. 
+* Cloud balancing: Assigning processes across computers with different specifications and costs. 
+
+.Prerequisites
+
+You have downloaded and unpacked the {PRODUCT} add-ons package, which includes the examples. For instructions about these actions, see <<examples-download-proc>>.
```

Comment:
You have downloaded and extracted the {PLANNER} examples from the {PRODUCT} `add-ons.zip` file. For instructions, see the download steps in <<ID for the consolidated "Downloading and running" proc that I suggested.>>.

Reply:
Consolidating two procedures into one then referencing *some of the steps* does not sound like a good decision to me.

Reply:
Agreed. Overly breaking down procedures didn't seem great to me either. So just gains and losses of both.